package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "react-accordion",
+  "version": "1.0.0",
+  "description": "A really lightweight, symantic, super simple accordion for React",
+  "main": "src/index.tsx",
+  "scripts": {
+    "develop": "servatron -d dist & esbuild --watch --bundle demo/index.tsx --outfile=dist/index.min.js --loader:.js=jsx",
+    "build": "esbuild --bundle src/index.tsx --outfile=dist/index.min.js --loader:.js=jsx",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/votemike/react-accordion.git"
+  },
+  "keywords": [
+    "accordion",
+    "accessible",
+    "react",
+    "ui"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/votemike/react-accordion/issues"
+  },
+  "homepage": "https://github.com/votemike/react-accordion#readme",
+  "dependencies": {
+    "react": "^18.2.0"
```

Comment:
We want to support a few versions of React don't we? Not just the latest version.
Should this be a peer dependency?